pyproject.toml

```diff
@@ -38,6 +38,7 @@ prettytable = "^3.9.0"
 semver = "^3.0.2"
 tomlkit = "^0.12.2"
 checkov = "^3.1.67"
+pytest-xdist = "^3.5.0"
```

Comment:
This should be moved into the `dev` dependencies section which is right below this one.